File: tests/test_lionshare.py
from lionshare import Lionshare

def test_prices():
    """Tests an API call to get coin prices"""

    lionshare = Lionshare()
    response = lionshare.get_prices('hour')

    assert isinstance(response, dict)
    assert isinstance(response['data'], dict)

def test_markets():
    """Tests an API call to get coin markets"""

    lionshare = Lionshare()
    response = lionshare.get_markets()

    assert isinstance(response, dict)
    assert isinstance(response['data'], dict)

File: js/test/testLionshare.js
'use strict';

var expect = require('chai').expect;
const Lionshare = require('../index');

describe('#Lionshare API', () => {
  it('should get coin prices', async () => {
    const client = new Lionshare();
    var response = await client.getPrices('hour');
    expect(response).to.be.an('object');
    expect(response['data']).to.be.an('object');
  });

  it('should get coin markets', async () => {
    const client = new Lionshare();
    var response = await client.getMarkets();
    expect(response).to.be.an('object');
    expect(response['data']).to.be.an('object');
  });
});